src/test/java/seedu/address/model/person/ContainsGeneralKeywordsPredicateTest.java

```diff
@@ -12,21 +12,24 @@
 
 import seedu.address.testutil.PersonBuilder;
 
-public class NameContainsKeywordsPredicateTest {
+public class ContainsGeneralKeywordsPredicateTest {
 
     @Test
     public void equals() {
         List<String> firstPredicateKeywordList = Collections.singletonList("first");
         List<String> secondPredicateKeywordList = Arrays.asList("first", "second");
 
-        NameContainsKeywordsPredicate firstPredicate = new NameContainsKeywordsPredicate(firstPredicateKeywordList);
-        NameContainsKeywordsPredicate secondPredicate = new NameContainsKeywordsPredicate(secondPredicateKeywordList);
+        ContainsGeneralKeywordsPredicate firstPredicate =
```

Comment:
Good to see you renamed the NameContainsKeywordsPredicate class to suit current context